subgraphs/across-v2/src/handlers/liquidity.ts

```diff
@@ -72,6 +78,20 @@ export function handleLiquidityAdded(event: LiquidityAdded): void {
   const amount = event.params.amount;
   const account = sdk.Accounts.loadAccount(event.params.liquidityProvider);
   account.liquidityDeposit(pool, amount);
+
+  // tvl
+  let inputTokenBalance: BigInt;
+  const erc20 = _ERC20.bind(event.params.l1Token);
+  const inputTokenBalanceResult = erc20.try_balanceOf(event.address);
+  if (inputTokenBalanceResult.reverted) {
+    log.info(
+      "[ERC20:balanceOf()]calculate token balance owned by bridge contract reverted",
+      []
+    );
+  } else {
+    inputTokenBalance = inputTokenBalanceResult.value;
+  }
+  pool.setInputTokenBalance(inputTokenBalance!);
```

Comment:
We can't assume the inputTokenBalance here. In the reverted block we should make the default value 0, or initialize the variable to 0 to start

Reply:
agreed